src/hello_moveit_interfaces/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ find_package(rosidl_default_generators REQUIRED)
 # Generate interfaces for actions
 rosidl_generate_interfaces(${PROJECT_NAME}
   "action/PickPlaceRepeat.action"
+  "action/SimpleActionMsg.action"
```

Comment:
Make this more descriptive.

Reply:
9f696b7d2d205fd2038f6a764db896805352b3fb has a new msg file pdf_beamtime_control_msg.action